perftracker/views.py

```diff
@@ -38,7 +36,10 @@
     PTRegressionServSideView
 from perftracker.models.test import TestModel, TestSimpleSerializer, TestDetailedSerializer
 from perftracker.models.test_group import TestGroupModel, TestGroupSerializer
+from perftracker.models.train_data import TrainDataModel, CHART_FUNCTION_TYPES, \
+    CHART_OUTLIERS, CHART_OSCILLATION, CHART_ANOMALY
 from perftracker.rest import pt_rest_ok, pt_rest_err, pt_rest_not_found, pt_rest_method_not_allowed
+from perftracker_django.settings import DEBUG
```

Comment:
don't think DEBUG is the right one.
my proposal was to add new flag that enables your logic